internal/pkg/pluginmanager/manager.go

```diff
@@ -66,29 +82,98 @@ func CheckLocalPlugins(conf *configloader.Config) error {
 
 	for _, tool := range conf.Tools {
 		pluginFileName := configloader.GetPluginFileName(&tool)
+		pluginMD5FileName := configloader.GetPluginMD5FileName(&tool)
 		if _, err := os.Stat(filepath.Join(pluginDir, pluginFileName)); errors.Is(err, os.ErrNotExist) {
 			if err != nil {
 				return err
 			}
 			return fmt.Errorf("plugin %s doesn't exist", tool.Name)
 		}
+		if _, err := os.Stat(filepath.Join(pluginDir, pluginMD5FileName)); errors.Is(err, os.ErrNotExist) {
+			if err != nil {
+				return err
+			}
+			return fmt.Errorf(".md5 file of plugin %s doesn't exist", tool.Name)
```

Comment:
? why return the err when `err != nil` and return another err when `err==nil`

Reply:
This is left over from the original code, I changed it